### ISSUE TITLE
IE11 - Add tabindex="-1" to On This Page headings so they will function properly

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3580]
## Description

IE11 does not honor in-page anchor links if the item being "anchored" to does not have a `tabindex=-1"`. I tested this with JAWS and the page would usually bounce back to the top after clicking the link, which was not correct behavior. Screen shot attached.

## Related Issue
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/3160

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Screenshots or Trace Logs

![Screen Shot 2019-11-19 at 1.52.33 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/8f75fd28-ba03-4bb1-a7a4-c5d84585e749)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/11139
https://github.com/department-of-veterans-affairs/vagov-content/pull/599

**Comment by delli-gatti-michael** [comment]
Passes local testing

**Comment by 1Copenut** [comment]
@delli-gatti-michael Thank you for the follow up. LMK when it's in staging and we'll spot test quick and close.

**Comment by desireeturner** [comment]
@1Copenut this is now on staging.

**Comment by 1Copenut** [comment]
@desireeturner I verified this on Staging with IE11. Moving to close.
